tests/testthat/test-parsnip-case-weights.R

```diff
@@ -0,0 +1,1126 @@
+skip_if_not_installed("parsnip", "0.2.1.9001")
+skip_if_not_installed("hardhat", "0.2.0.9000")
+skip_if_not_installed("yardstick", "0.0.9.9000")
+skip_if_not_installed("workflows", "0.2.6.9001")
+skip_if_not_installed("recipes", "0.2.0.9001")
+skip_if_not_installed("discrim", "0.2.0.9000")
+skip_if_not_installed("multilevelmod", "0.1.0.9000")
+skip_if_not_installed("poissonreg", "0.2.0.9000")
+skip_if_not_installed("rules", "0.2.0.9000")
+skip_if_not_installed("baguette", "0.2.0.9000")
+skip_if_not_installed("censored", "0.0.0.9000")
+skip_if_not_installed("poissonreg", "0.2.0.9000")
+
+
+library(parsnip)
+library(censored)
+library(baguette)
+library(rules)
+library(poissonreg)
+library(multilevelmod)
+library(discrim)
+library(sparklyr)
+
+# ------------------------------------------------------------------------------
+
+expect_unequal <-
+  function(object, expected, ...,
+           tolerance = if (edition_get() >= 3) testthat_tolerance()) {
+    expect_true(!compare(object, expected, tolerance = tolerance, ...)$equal)
+  }
+
+# ------------------------------------------------------------------------------
+# bagged trees
+
+test_that('bag_tree - rpart case weights', {
+  dat <- make_two_class_wts()
+
+  expect_error({
+    set.seed(1)
+    wt_fit <-
+      bag_tree() %>%
+      set_engine("rpart") %>%
+      set_mode("classification") %>%
+      fit(Class ~ ., data = two_class_dat, case_weights = dat$wts)
+  },
+  regexp = NA)
+
+  set.seed(1)
+  unwt_fit <-
+    bag_tree() %>%
+    set_engine("rpart") %>%
+    set_mode("classification") %>%
+    fit(Class ~ ., data = two_class_dat)
```

Comment:
I changed this line to use the full dataset instead of the subset since the expectation below is about unequal fits.